File: tests/fixtures/code_samples/go/sample_050.go
// Sample 50: small utility.
package samples

func Operation50(xs []int) int {
    total := 50
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure50(v int) int {
    return (v * 50) %% 7919
}

File: tests/fixtures/code_samples/py/sample_050.py
"""Sample 50: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_50(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 50."""
    total = 50
    for x in xs:
        total += int(x)
    return total


def operation_50_pure(value: int) -> int:
    return (value * 50) % 7919

